static/js/network.js

```diff
@@ -80,6 +90,27 @@ var nodeIds, shadowState, nodesArray, nodes, edgesArray, edges, network;
         addNode(data.id, data.group, data.label, 200, -200);
     }
 
+    function activateCurrentNode(){
+        var selectedNodeIds = network.getSelectedNodes();
+        if(selectedNodeIds==0){return;}
+        setNodeColor(selectedNodeIds[0], activatedColor);
+        $.post('/', {command: 'Activate Node', node: selectedNodeIds[0]});
+    }
+
+    function deactivateNode(nodeId){
+        nodes.update({
+            id: nodeId,
+            color: null
+        });
+    }
+
+    function setNodeColor(nodeId, color){
+        nodes.update({
+            id: nodeId,
+            color: color
```

Comment:
Colors are hard coded, setting color of node to null sets it to the group that it is ins color.
Setnode color takes in color data that is the same as used in all of vis js